datalad/distribution/create_sibling.py

```diff
@@ -287,6 +302,21 @@ def _create_dataset_sibling(
     return remoteds_path
 
 
+def _ls_remote_path(ssh, path):
```

Comment:
candidate for commands abstractions: https://github.com/datalad/datalad/pull/4068/files#r379009046

Reply:
True. Will integrate (I'm on it anyway)